src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -110,6 +110,12 @@ private TestKitBase(ITestKitAssertions assertions, ActorSystem system, Config co
             {
                 InternalCurrentActorCellKeeper.Current = (ActorCell)((ActorRefWithCell)testActor).Underlying;
             }
+            else if(!(this is TestProbe)) 
```

Comment:
Could you put a //TODO: on this line, letting someone in the future know why it's there?